### ISSUE TITLE
Modern Input Widgets

### ISSUE THREAD

**Issue by gotwig** [BabylonJS/Editor#95]
Modern Input Widgets should be used by the editor for enhanced accessibility and space saving.

Compare for example with unreal engine 4:

https://docs.unrealengine.com/portals/0/images/GettingStarted/SubEditors/editortypes_topic.png

Seperate widgets for X,Y,Z input - Such values should be displayed in one line instead of 3.

Dedicated Color Picker only showing the color. A click opens the color picker where users can set R,G,B (A) - this is not possible with HTML5.

**Comment by julien-moreau** [comment]
Just added native color picker of browsers
Do you think you can perform the same result with dat.gui for vectors and colors?
